### PR TITLE
Add internal slots to MediaStreamTrack.

### PR DESCRIPTION
**Pull request by jan-ivar**
Fixes https://github.com/w3c/mediacapture-main/issues/812.

***
<a href="https://pr-preview.s3.amazonaws.com/jan-ivar/mediacapture-main/pull/822.html" title="Last updated on Dec 2, 2021, 8:54 PM UTC (893e1d7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/mediacapture-main/822/371cccd...jan-ivar:893e1d7.html" title="Last updated on Dec 2, 2021, 8:54 PM UTC (893e1d7)">Diff</a>

### CODE REVIEW

**Comment by jan-ivar** [comment]
Note this PR is on top of https://github.com/w3c/mediacapture-main/pull/821.

File: getusermedia.html [hunk 1]
```diff
@@ -1105,7 +1158,8 @@ <h2>Methods</h2>
                     {{MediaStreamTrack}} object.</p>
                   </li>
                   <li>
-                    <p>If <var>track</var>.{{MediaStreamTrack/readyState}}
+                    <p>If <var>track</var>'s <a data-link-for="MediaStreamTrack"
+                      data-link-type="attribute">[[\ReadyState]]</a>
```
Comment (alvestrand) [inline getusermedia.html:1162]: @dontcallmedom is there a more compatct way of writing internal-slot references?

Comment (dontcallmedom) [reply getusermedia.html:1162]: (assuming the internal slot has been properly `<dfn>`d, which I haven't checked)

**Comment by jan-ivar** [comment]
Rebased on top of tip to fix errors.

**Pull request merged**